src/expressApp.js

```diff
@@ -117,7 +117,7 @@ function configureProxy(app, { env }) {
   });
 
   app.use(function(req, res) {
-    if (req.url && /^\/api\/v[12]\/.*$/.test(req.url)) {
+    if (req.url && (/^\/api\/v[12]\/.*$/.test(req.url) || /^\/oauth\/token.*$/.test(req.url))) {
       req.isProxy = true;
```

Comment:
i'm sure there's a way to do this in 1 regex instead of 2 but I'm not a regex wiz :( haha